data-prepper-plugins/peer-forwarder/src/test/java/com/amazon/dataprepper/plugins/processor/peerforwarder/PeerForwarderTest.java

```diff
@@ -95,45 +115,34 @@ public void testLocalIpOnly() {
         Assert.assertTrue(expectedResourceSpans.containsAll(exportedResourceSpans));
     }
 
-    @Test
-    public void testProcessRequest() throws Exception {
-        final PeerForwarder testPeerForwarder = generatePeerForwarder(Collections.singletonList(LOCAL_IP), 2);
-        final TraceServiceGrpc.TraceServiceBlockingStub mockClient = mock(TraceServiceGrpc.TraceServiceBlockingStub.class);
-        doReturn(null).when(mockClient).export(any());
-        final List<Record<ExportTraceServiceRequest>> localBuffer = new ArrayList<>();
-        final Object[] args1 = new Object[]{null, REQUEST_1, localBuffer};
-        Whitebox.invokeMethod(testPeerForwarder, "processRequest", args1);
-        Assert.assertEquals(1, localBuffer.size());
-        Assert.assertEquals(REQUEST_1, localBuffer.get(0).getData());
-        final Object[] args2 = new Object[]{mockClient, REQUEST_2, localBuffer};
-        Whitebox.invokeMethod(testPeerForwarder, "processRequest", args2);
-        Assert.assertEquals(1, localBuffer.size());
-        verify(mockClient, times(1)).export(REQUEST_2);
-    }
-
     @Test
     public void testSingleRemoteIpBothLocalAndForwardedRequest() throws Exception {
         final List<String> testIps = generateTestIps(2);
-        final PeerForwarder testPeerForwarder = spy(generatePeerForwarder(testIps, 3));
+        final Channel channel = mock(Channel.class);
+        final TraceServiceGrpc.TraceServiceBlockingStub client = mock(TraceServiceGrpc.TraceServiceBlockingStub.class);
```

Comment:
For mock objects consistent across multiple tests, could factor out by declaring as instances variables like:
```
@Mock
private Channel channel;
...
@Mock 
private TraceServiceGrpc.TraceServiceBlockingStub client;
```
The `@RunWith(MockitoJUnitRunner.class)` you have at the top will automatically call `initMocks()` and create mock objects where it sees the `@Mock` annotation.

Reply:
Good catch. Should also globalize client instance